src/modules/BootSupportCheck.rb

```diff
@@ -174,26 +176,39 @@ def check_mbr
       false
     end
 
+    def check_tpm2
+      return true unless Y2Storage::StorageManager.instance.encryption_use_tpm2
+
+      add_new_problem(_("This bootloader cannot handle encryption supported by a TPM2 device. "))
```

Comment:
so even our patched grub2 cannot handle tpm2 device?

Reply:
The text is confusing. Sure grub2 handle TPM2 devices since very long ago. I guess this text will not be present when EncryptionMethod is used.

Reply:
JFI: "EncryptionMethod" ....This will be a new PR based on the current.